src/inputs/buttonConstants.js

```diff
@@ -3,24 +3,63 @@ import Utils from '../utils/utils';
 
 export const buttonPropTypes = {
     as: PropTypes.oneOf(['a', 'button']),
+    /**
+     * Primary content.
+     */
     children: PropTypes.node.isRequired,
+    /**
+     * Additional classes.
+     */
     className: PropTypes.string,
+    /**
+     * Color of the button.
+     */
     color: PropTypes.oneOf(Utils.colorEnums()),
+    /**
+     * A button can reduce its padding.
+     */
     compact: PropTypes.bool,
+    /**
+     * A button can be disabled.
+     */
     disable: PropTypes.bool,
+    /**
+     * Deprecated prop.
+     */
     disabled: PropTypes.bool,
+    /**
+     * A button can stretch the width of it's container.
```

Comment:
Should be 'its' (no apostrophe).
Also, does `fluid` mean the thing 'stretches the width of its container'?  Or more something along the lines of the thing 'takes up the full width of its container'?

Reply:
Fixed.

Reply:
The change to "The Button will be resized to its parent container's width" in 170fcf6 seems good to me.  Resolving,